config/routes.rb

```diff
@@ -1,5 +1,6 @@
 Rails.application.routes.draw do
```

Comment:
A PR must contain one new _feature_ and whatever code is needed to implement the feature. A feature is described as a new functionality for a user -- "the user can now view X" or whatever -- and will inevitably involve creating/modifying the frontend.

Reply:
User features added, show an index page